packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -600,6 +600,19 @@ const ComboBox = forwardRef(
                   event.target.value.length
                 );
               }
+
+              if (event.altKey && event.key == 'ArrowDown') {
```

Comment:
```
if(match(event, keys.Alt) && match(event,keys.ArrowDown))
{
 ////
}

```
this doesn't seem to be working ??

Reply:
There is no definition for `keys.Alt` in `packages/react/src/internal/keyboard/keys.js`. Could you add it and retry using `match()` again?

Reply:
> ```
> if(match(event, keys.Alt) && match(event,keys.ArrowDown))
> {
>  ////
> }
> ```
> 
> this doesn't seem to be working ??

I had tried this after adding definition for Alt key
However, if I put any one condition it works as expected but both simultaneously are not working :(